src/NUnitFramework/framework/Internal/Commands/TimeoutCommand.cs

```diff
@@ -21,37 +21,40 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
-#if THREAD_ABORT
 using System;
-using System.Collections.Generic;
 using System.Threading;
+#if !THREAD_ABORT
+using System.Threading.Tasks;
+#endif
+using NUnit.Framework.Interfaces;
 
 namespace NUnit.Framework.Internal.Commands
 {
-    using Execution;
-    using Interfaces;
-
     /// <summary>
     /// TimeoutCommand creates a timer in order to cancel
     /// a test if it exceeds a specified time and adjusts
     /// the test result if it did time out.
     /// </summary>
     public class TimeoutCommand : BeforeAndAfterTestCommand
     {
+        private readonly int _timeout;
+#if THREAD_ABORT
         Timer _commandTimer;
-        private bool _commandTimedOut = false;
+        private bool _commandTimedOut;
+#endif
 
         /// <summary>
         /// Initializes a new instance of the <see cref="TimeoutCommand"/> class.
         /// </summary>
         /// <param name="innerCommand">The inner command</param>
         /// <param name="timeout">Timeout value</param>
-        public TimeoutCommand(TestCommand innerCommand, int timeout)
-            : base(innerCommand)
+        public TimeoutCommand(TestCommand innerCommand, int timeout) : base(innerCommand)
```

Comment:
ℹ Note for the future: we try to not reformat code unless the existing formatting was way off. It keeps the git blame history cleaner for those lines. Same with the string.Format below. To make improvements that aren't intrinsically related to this PR, it's nice to do it in a separate PR. It's less "all-or-nothing" and is easier on reviewers.